### ISSUE TITLE
Conflict with `uvesafb-dkms`. And remove `uvesafb`-dependency.

### ISSUE THREAD

**Issue by dreirund** [bartoszek/AUR-v86d#1]
Dear 'bartus',

you maintain the AUR-package "v86d":
https://aur.archlinux.org/packages/v86d/

There is already about one year old an issue reported which still is
no resolved:

> There is a conflict with uvesafb-dkms which (now?) also provides v86d.
> 
> So installing v86d will also install uvesafb-dkms which will cause the
> following error:
> 
> error: failed to commit transaction (conflicting files) v86d:
> /usr/bin/v86d exists in filesystem (owned by uvesafb-dkms) v86d:
> /usr/lib/initcpio/hooks/v86d exists in filesystem (owned by
> uvesafb-dkms) v86d: /usr/lib/initcpio/install/v86d exists in filesystem
> (owned by uvesafb-dkms) v86d: /usr/lib/modprobe.d/uvesafb.conf exists
> in filesystem (owned by uvesafb-dkms) Errors occurred, no packages were
> upgraded.

Best idea to resolving is not to depend on any `uvesafb`-package, since
`v86d` _might_ be also wanted to used for other hacking. (In fact, `v86d`
is a helper vor `uvesafb`, not the other way round) or a custom kernel
might already have `uvesafb` compiled in.

And, please take care of proble reports on the AUR-page and not just
refer to somewhere else. You can configure the AUR to send you emails,
when someone reports something. And you cannot expect of bug-reporters to have an github-account. Do not make hindrances for bug-reporters. And, in https://aur.archlinux.org/packages/v86d/#pinned-689607 you mentioned an email address (`v86d (at) github.com`) where to report issues to, but that email address does not work ("The email account that you tried to reach does not exist."). Please fix.

**Comment by felixbecker2** [comment]
Dear 'bartus',

you maintain the AUR-package "v86d":
https://aur.archlinux.org/packages/v86d/

There is already about one year old an issue reported which still is
no resolved:

> There is a conflict with uvesafb-dkms which (now?) also provides v86d.
> 
> So installing v86d will also install uvesafb-dkms which will cause the
> following error:
> 
> error: failed to commit transaction (conflicting files) v86d:
> /usr/bin/v86d exists in filesystem (owned by uvesafb-dkms) v86d:
> /usr/lib/initcpio/hooks/v86d exists in filesystem (owned by
> uvesafb-dkms) v86d: /usr/lib/initcpio/install/v86d exists in filesystem
> (owned by uvesafb-dkms) v86d: /usr/lib/modprobe.d/uvesafb.conf exists
> in filesystem (owned by uvesafb-dkms) Errors occurred, no packages were
> upgraded.

-- >% --

Please resolve the issue, or I will flag your package as out-of-date
because you seem not to maintain it.

Best idea to resolving is not to depend on any `uvesafb`-package, since
`v86d` _might_ be also wanted to used for other hacking. (In fact, `v86d`
is a helper vor `uvesafb`, not the other way round) or a custom kernel
might already have `uvesafb` compiled in.

And, please take care of proble reports on the AUR-page and not just
refer to somewhere else. You can configure the AUR to send you emails,
when someone reports something.

**Comment by bartoszek** [comment]
I've stripped the PKGBUILD to only provide the `v86d` emulator binary.

You've got a point there - I got carried away when making a [script to mass comment on AUR](https://github.com/bartoszek/aur-post). Using `@` sign in GitHub link is confusing, and someone may mistaken it for an actual email adress :thinking:

Let me also rebuttal your statement `that having user post issue on github is to hassle`. I find this a weak argument. Creating a GitHub account is like two clicks using your Google credentials, and scope of possibilities to contribution makes this a well invested efert. Users can create fast changes using just GitHub interface an maintainers cans simply apply/reject/review those without leaving their platform. [ihmo] Arch should just ditch AUR web all together and come to GitHub - e.g. Manjaro.

Cheers, and thanks for your contribution :blush:
